pkg/raft/raft_paper_test.go

```diff
@@ -132,6 +132,8 @@ func TestLeaderBcastBeat(t *testing.T) {
 			sort.Sort(messageSlice(msgs))
 			if storeLivenessEnabled {
 				assert.Equal(t, []pb.Message{
+					{From: 1, To: 2, Term: 1, Type: pb.MsgApp, Entries: r.raftLog.allEntries()},
```

Comment:
Could you update the test description?

Also, consider putting a `require.Empty(t, r.readMessages())` before the ticks. Maybe even after each tick until the last one. And so `hi > 1` would be good :)